swarm/api/http/server.go

```diff
@@ -589,6 +643,9 @@ func (s *Server) HandleGetFile(w http.ResponseWriter, r *Request) {
 }
 
 func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	requestCount.Inc(1)
+	startTime := time.Now()
```

Comment:
This is exactly what I am doing 2 lines below. I have kept both timers so that we illustrate again the difference. In the future we can get rid of the default timer.